appengine/endpoints/app.yaml

```diff
@@ -11,7 +11,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-runtime: nodejs
+runtime: nodejs12
```

Comment:
Thanks for the fix!  I'm a little surprised this was failing with the current `app.yaml`, because this one is using `env: flex`.  The rules for valid runtimes (at least last time I checked) were different here.  

Just to double check - are you using `env: flex` in your `app.yaml`?

Reply:
Good catch, no, I was using `env: standard`.